src/resources/InsightPanelInterfaces/InsightPanelInterface.model.ts

```diff
@@ -14,9 +15,7 @@ export enum InsightPanelResultTemplateLayout {
     Thumbnail = 'thumbnail',
 }
 
-interface InsightPanelOption {
-    enabled: boolean;
-}
+interface InsightPanelOption extends HostedInterfaceOption {}
```

Comment:
@nathanlb I'm pretty sure it wouldn't be breaking if I add the `useSmartSnippet` option in the InsightPanel interface. Do you want me to add it now?

Reply:
The normal flow requires the changes to be live in production before we can merge a PR like this as the client interfaces need to stay in sync with what's there.
So you need to deploy the change for IPX before merging this and if you include insight panel we need to have it done as well. We're planning on doing it this sprint but we're at least a few days away from having it in prod.